src/components/Cart/CartProducts/ProductsDetails.vue

```diff
@@ -45,6 +53,9 @@ export default defineComponent({
         .toUpperCase()
     );
 
+    onUpdated(() => {
```

Comment:
I suggest using `watch` or `watchEffect` instead of `onUpdated` hook, it's mostly used for debugging